app/src/main/java/com/github/se/polyfit/di/UserModule.kt

```diff
@@ -22,64 +21,60 @@ import javax.inject.Singleton
 @InstallIn(SingletonComponent::class)
 object UserModule {
 
-  @Provides
-  @Singleton
-  fun providesUser(): User {
-    // this is just a safegaured, the user should be signed in before this is called
-    // if for some reason the user is not signed in, without this, it could cause and error
-    // firestoredatabase needs a user id to be initialized
-    return User(id = "testUserID")
-  }
+    @Provides
+    @Singleton
+    fun providesUser(): User {
+        // this is just a safegaured, the user should be signed in before this is called
+        // if for some reason the user is not signed in, without this, it could cause and error
+        // firestoredatabase needs a user id to be initialized
+        return User(id = "testUserID")
+    }
```

Comment:
The IDE is wrongly setup for the tabs and space. May be corrected with ktfmt check